variable/test/test_variables.cpp

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Simon Heybrock
+#include <gtest/gtest.h>
+
+#include "test_variables.h"
+
+using namespace scipp;
+
+INSTANTIATE_TEST_SUITE_P(
+    Scalar, DenseVariablesTest,
+    testing::Values(makeVariable<double>(Values{1.2}),
+                    makeVariable<double>(Values{1.2}, Variances{1.3}),
+                    makeVariable<float>(Values{1.2}, units::m),
+                    makeVariable<int64_t>(Values{12}),
+                    makeVariable<int32_t>(Values{4}, units::s),
+                    makeVariable<std::string>(Values{"abc"})));
```

Comment:
Because variables are uninitialised in `empty_like` you are not testing values - but would it be advisable to try to keep the Variable values the same between the different types for other test usage?

Reply:
As stated in the PR message this set of test data is incomplete. I think we may want to include even more different values.